src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
```

Comment:
The project is missing the `App.js` component which is the main component. Please add `App.js` in the `src` directory and import `Calculator.js` to it.

Reply:
Hi, so should I import Calculator.js into App.js and then App.js into index.js?

Reply:
I do not know what the purpose of index.js is if App.js is the main component